src/cpu/aarch64/acl_winograd_convolution.cpp

```diff
@@ -109,7 +109,7 @@ status_t acl_wino_convolution_fwd_t::pd_t::init_conf() {
 
     const bool shape_ok
             // only unit strides allowed
-            = (acp_.padstride_info.stride() == std::pair<uint, uint> {1, 1})
```

Comment:
How is this change related to CI?

Reply:
The GCC workflow fails otherwise. Something to do with `uint` not defined.

Reply:
No problem!